app/models/micropost.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope -> { order(created_at: :desc) }
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
+  validates :content, presence: true, length: { maximum: 140 }
+  validate  :picture_size
```

Comment:
```suggestion
  validate :picture_size
```
validateのあとのスペース一個多そうですね🤔

Reply:
空白を一つにしました。